src/pages/SvarSed/SEDNew.tsx

```diff
@@ -414,7 +415,10 @@ const SEDNew = (): JSX.Element => {
                   dispatch(appActions.appReset()) // cleans person and sak reducer
                 }
               }}
-              onPersonFound={() => setIsFnrValid(true)}
+              onPersonFound={() => {
+                setIsFnrValid(true)
+                setGrading(person?.adressebeskyttelse != undefined ? person.adressebeskyttelse : "")
```

Comment:
Hvis grading kan være NULL (se kommentar linje 212) - og trenger du sjekke på undefined, hva hvis NULL. Følgernde vil vel funke for både undefined og null:
`person?.adressebeskyttelse  ? person.adressebeskyttelse : null`